.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Run the checkrefs.py script and check for typos.
+on:
+  push:
+    branches: [master]
```

Comment:
Path filters needed here so the workflow will only run when relevant files are changed